spec/cog_cmd/ecs/service/show_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe 'the service-show command' do
+  let(:command_name) { 'service-show' }
+
+  let(:client) { Object.new }
```

Comment:
An object here is probably fine, but you might want to use a real test double: `double("Aws::ECS::Client")` just so you get better error messages when you try to do something with it that it wasn't expecting.